### PR TITLE
fixed wbtc display bug

### PR DESCRIPTION
**Pull request by kendricktan**
turns out wbtc has 8 decimals

### CODE REVIEW

File: packages/frontend/containers/Coins.tsx [hunk 1]
```diff
@@ -32,6 +35,7 @@ const COINS = {
     symbol: "USDC",
     icon: "Usd",
     stable: true,
+    decimals: 8,
```
Comment (adrianmcli) [inline packages/frontend/containers/Coins.tsx:38]: @kendricktan I thought USDC was 6 decimals?

File: packages/frontend/features/swap-tokens/useSwap.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ const useSwap = (thingToSwap, fromTokenStr, toTokenStr, amountToSwap) => {
     const { cEther, cDai, cBat, cUsdc, cRep, cZrx, cWbtc } = contracts;
 
     const amount: Wei =
-      fromTokenStr === "usdc" ? inWei(amountToSwap, 6) : inWei(amountToSwap);
+      fromTokenStr === "usdc" || fromTokenStr === "wbtc" ? inWei(amountToSwap, 6) : inWei(amountToSwap);
```
Comment (adrianmcli) [inline packages/frontend/features/swap-tokens/useSwap.ts:19]: Is it 6 or 8 decimals?

File: packages/frontend/containers/Coins.tsx (hunk 1 above)
Comment (kendricktan) [reply packages/frontend/containers/Coins.tsx:38]: Not the best practice but i force pushed to master https://github.com/studydefi/dedge/commit/ff46a7048c5c5bde80dc123091068f6c5fd5fc1e

File: packages/frontend/features/swap-tokens/useSwap.ts (hunk 2 above)
Comment (kendricktan) [reply packages/frontend/features/swap-tokens/useSwap.ts:19]: I'll make a new fix

**Pull request merged**